pkg/command/hello.go

```diff
@@ -57,8 +56,8 @@ func HelloHandler(ctx context.Context, cancel context.CancelFunc, gr run.Group,
 
 		gr.Add(func() error {
 			return server.Run()
-		}, func(_ error) {
-			logger.Info().
+		}, func(err error) {
```

Comment:
This error includes the reason for shutdown, I have not assigned it to the log because even a Ctrl+C gets didsplayed as an error which feels strange.